File: net/net-registri-log/AuditLog/Models/FiltriAuditLog.cs
using System;

namespace net_registri_log.AuditLog.Models
{
    public class FiltriAuditLog
    {
        //public DateTime? DataDa { get; set; }
        //public DateTime? DataA { get; set; }
        public string UserId { get; set; }
        public string Type { get; set; }
        public string TableName { get; set; }
        public bool Desc { get; set; }
        public string OrderColumn { get; set; }
    }
}

File: node/projects/ngx-registri-log/src/lib/audit-log/filtri-audit-log.ts
import { Pagination } from '../shared/pagination';

export class FiltriAuditLog {
    userId: string;
    type: string;
    tableName: string;

    dataDa: Date | null;
    dataA: Date | null;

    paginazione: Pagination;
    desc: boolean;
    orderColumn: string;
    
    constructor(data?: Partial<FiltriAuditLog>) {
        Object.assign(this, data);
        this.dataDa =  new Date(this.dataDa);
        this.dataA =  new Date(this.dataA);
    }
}